src/routes/partner/+page.svelte

```diff
@@ -51,84 +79,179 @@
 		} else {
 			maxLength = 99;
 		}
-	});
 
-	function truncateString(str) {
-		if (str.length > maxLength) {
-			return str.slice(0, maxLength) + '...';
-		} else {
-			return str;
-		}
-	}
+		map = new maplibregl.Map({
+			container: 'map',
+			style:
+				'https://vectortiles.geo.admin.ch/styles/ch.swisstopo.leichte-basiskarte.vt/style.json', //'https://basemaps.cartocdn.com/gl/positron-gl-style/style.json',
+			center: [7.59249, 47.55654], // starting position
+			zoom: 14, // starting zoom;
+			maxBounds: [
+				[5.94, 45.81],
+				[10.51, 47.81]
+			],
+			minZoom: 12,
+			maxZoom: 20,
+			scrollZoom: true,
+			attributionControl: true
+		});
+
+		// Adding scale bar to the map
+		let scale = new maplibregl.ScaleControl({
+			maxWidth: 200,
+			unit: 'metric'
+		});
+		map.addControl(scale, 'bottom-left');
+
+		// Adding zoom and rotation controls to the map
+		map.addControl(new maplibregl.NavigationControl());
+
+		const selectInputControl = new SelectInputControl(features, map);
+		map.addControl(selectInputControl, 'top-left');
+
+		map.on('load', async () => {
+			try {
+				const image = await map.loadImage('./src/lib/data/pin-48.png');
+				map.addImage('custom-marker', image.data);
+
+				map.addSource('collaborators', {
+					type: 'geojson',
+					data: data,
+					cluster: true,
+					clusterMaxZoom: 15, // Max zoom to cluster points on
+					clusterRadius: 60 // Radius of each cluster when clustering points (defaults to 50)
+				});
+
+				map.addLayer({
+					id: 'clusters',
+					type: 'circle',
+					source: 'collaborators',
+					filter: ['has', 'point_count'],
+					paint: {
+						// Use step expressions (https://maplibre.org/maplibre-style-spec/#expressions-step)
+						// with three steps to implement three types of circles:
+						//   * Blue, 20px circles when point count is less than 100
+						//   * Yellow, 30px circles when point count is between 100 and 750
+						//   * Pink, 40px circles when point count is greater than or equal to 750
+						'circle-color': '#70416C',
+						'circle-radius': ['step', ['get', 'point_count'], 20, 3, 30, 5, 40]
+					}
+				});
+
+				map.addLayer({
+					id: 'cluster-count',
+					type: 'symbol',
+					source: 'collaborators',
+					filter: ['has', 'point_count'],
+					layout: {
+						'text-field': '{point_count_abbreviated}',
+						'text-font': ['Frutiger Neue Bold'],
+						'text-size': 20,
+						'text-offset': [0, 0.15]
+					},
+					paint: {
+						'text-color': '#FFFFFF'
+					}
+				});
+
+				// Add a symbol layer
+				map.addLayer({
+					id: 'collaborators',
+					type: 'symbol',
+					source: 'collaborators',
+					filter: ['!', ['has', 'point_count']],
+					layout: {
+						'icon-image': 'custom-marker',
+						'icon-overlap': 'always',
+						'icon-size': 1,
+						'text-field': ['get', 'label'],
+						'text-font': ['Frutiger Neue Bold'],
```

Comment:
@koilebeit Please do use our default font 'Euclid Circular B' as defined in https://github.com/Stadt-Geschichte-Basel/stadtgeschichtebasel.ch/blob/main/src/app.postcss

Reply:
Only fonts that are defined as glyphs in the style (https://vectortiles.geo.admin.ch/styles/ch.swisstopo.leichte-basiskarte.vt/style.json) can be used.

From a cartographic point of view, this also makes sense.

Whether and how custom fonts can be integrated is not yet clear to me.

Reply:
Makes total sense! Sorry for bringing it up